QUICKSTART.md

```diff
@@ -0,0 +1,45 @@
+# Kind
+This guide was created to help new users of Kubeflow install and test Kubeflow Helm Charts locally based on the kind.
+
+## Requirements
+* kind v0.25.0 or above
+* helm v3.16.3 or above
+* curl 8.7.1 or above
```

Comment:
I don't think we need any specific feature of `curl`. Maybe we can write that just the `curl` itself is required for quickstart?